.github/workflows/check-cves.yaml

```diff
@@ -0,0 +1,27 @@
+name: check cves
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  cves-check:
+    uses: canonical/robotics-actions-workflows/.github/workflows/cves-check.yaml@feature/cve-check
+    with:
+      snap-name: turtlebot3c
```

Comment:
We should have those example snaps on the store and change this for those snaps.

Could we then automatically trigger the snap workflow to rebuild/publish ?